app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  validates :user, presence: true
```

Comment:
We don't need to validate the presence of user and post anymore as our belongs_to associations take care of that for us